Python/sysmodule.c

```diff
@@ -1618,8 +1920,11 @@ sys_getandroidapilevel_impl(PyObject *module)
 #endif   /* ANDROID_API_LEVEL */
 
 
+
 static PyMethodDef sys_methods[] = {
     /* Might as well keep this in alphabetic order */
+    SYS_ADDAUDITHOOK_METHODDEF
+    {"audit",           sys_audit, METH_VARARGS, audit_doc },
```

Comment:
Maybe you could use METH_FASTCALL to avoid the temporary tuple to pass positional arguments.

Reply:
Do you have any docs on how to implement METH_FASTCALL that will help me here?